src/tests/ftest/util/daos_io_conf.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+import os
+from avocado.utils import process
+
+from apricot import TestWithServers
+from command_utils import ExecutableCommand, CommandFailure, FormattedParameter
+from command_utils import BasicParameter
+from test_utils import TestPool
+
+class IoConfGen(ExecutableCommand):
+    """Defines an object for the daos_gen_io_conf and daos_run_io_conf commands.
+
+    :avocado: recursive
+    """
+    def __init__(self, path="", env=None):
+        """Create a ExecutableCommand object.
+
+        Uses Avocado's utils.process module to run a command str provided.
+
+        Args:
+            command (str): string of the command to be executed.
+            path (str, optional): path to location of command binary file.
+                Defaults to ""
+        """
+        super(IoConfGen, self).__init__("/run/run_gen_io_conf/*",
+                                        "daos_gen_io_conf")
+        self.verbose = True
+        self.env = env
+        self.ranks = FormattedParameter("-g {}")
+        self.targets = FormattedParameter("-t {}")
+        self.obj_num = FormattedParameter("-o {}")
+        self.akeys = FormattedParameter("-a {}")
+        self.dkeys = FormattedParameter("-d {}")
+        self.re_size = FormattedParameter("-s {}")
```

Comment:
Something to consider:  Since this variable name becomes the key in the yaml file, I find it beneficial for the variable name to be descriptive enough that someone reading only the yaml file can generally understand what impact the setting has to the test.  In my opinion, I think something like `rec_size` or `record_size` would be easier to understand in this case.  Ultimately the user can cross reference the command help with the argument string assigned to this variable to determine what impact the values has, but a descriptive name could avoid that extra work.